oneview_redfish_toolkit/tests/blueprints/test_zone.py

```diff
@@ -125,6 +125,10 @@ def test_get_zone_when_uuid_is_only_template_id(self):
         ) as f:
             zone_without_drives_mockup = json.load(f)
 
+        zone_name_only_template_id = \
```

Comment:
The name of this variable seems weird because the name has not template_id, maybe should be `zone_name_only_with_template_name `, but all right with this, don't need change it.